### PR TITLE
date picker css fix

### PR DESCRIPTION
**Pull request by teryhill**
date picker css fix in Mips

### CODE REVIEW

**Comment by aethelwulffe** [comment]
Terry listed some things that still needed to be fixed.  Now I understand what that should be.

File: modules/MIPS/clinical_measures.php [hunk 1]
```diff
@@ -315,7 +318,28 @@ function Form_Validate() {
                     <td class='label'>
                       <?php echo htmlspecialchars(xl('Begin Date'), ENT_NOQUOTES); ?>:
                     </td>
```
Comment (aethelwulffe) [inline modules/MIPS/clinical_measures.php]: Oops.  Two begin dates.

Comment (tmccormi) [reply modules/MIPS/clinical_measures.php]: Should I revert this one?

**Comment by teryhill** [comment]
@aethelwulffe and I talked about it I did it in a hurry (had to get to the drying apples) he will fix it and send the fix.

**Comment by aethelwulffe** [comment]
Not totally sure it has to be fixed.  Currently, the case is always that 
the report type = 'pqrs'

The other condition is as to what is done when viewing a completed 
report, and I need to test this to see if it makes any real difference.

I will get that answer done one way or another.  In the meantime, no 
need to revert.

**Pull request merged**